apps/coscrad-frontend/src/components/resources/songs/song-index.presenter.tsx

```diff
@@ -24,7 +27,7 @@ export const SongIndexPresenter = ({ entities: songs }: SongIndexState) => {
             tableData={songs}
             cellRenderersDefinition={cellRenderersDefinition}
             heading={'Songs'}
-            filterableProperties={['lyrics', 'title', 'titleEnglish']}
+            filterableProperties={['lyrics', 'name']}
```

Comment:
remove 'lyrics' ?

Reply:
@jbambrick Can you make a story to put this back later? We don't necessarily have to show the lyrics in the table to make the song searchable by its lyrics, btw.